res/css/views/messages/_ViewSourceEvent.scss

```diff
@@ -35,13 +35,15 @@ limitations under the License.
         mask-size: auto 12px;
         visibility: hidden;
         background-color: $accent;
-        mask-image: url('$(res)/img/feather-customised/maximise.svg');
+        mask-image: url("$(res)/img/element-icons/maximise-expand.svg");
     }
 
     &.mx_ViewSourceEvent_expanded .mx_ViewSourceEvent_toggle {
         mask-position: 0 bottom;
         margin-bottom: 7px;
```

Comment:
For the updated icon, I believe this should be changed to 5px to ensure the centre of the icon is always under the cursor when toggling states.

```suggestion
        margin-bottom: 5px;
```